tests/build_system/test_pep_518.py

```diff
@@ -68,14 +68,15 @@ def test_load_build_system_pyproject(
 
     build_system = load_build_system(pex_project_dir)
     assert isinstance(build_system, BuildSystem)
-    assert "setuptools.build_meta" == build_system.build_backend
+    assert "pex_build.hatchling.build" == build_system.build_backend
     dists = {
         dist.metadata.project_name
         for dist in Virtualenv(build_system.venv_pex.venv_dir).iter_distributions()
     }
-    assert ProjectName("setuptools") in dists
+    assert ProjectName("hatchling") in dists
     subprocess.check_call(
-        args=[build_system.venv_pex.pex, "-c", "import {}".format(build_system.build_backend)]
+        args=[build_system.venv_pex.pex, "-c", "import {}".format(build_system.build_backend)],
+        env=build_system.env,
```

Comment:
This env carries the backend-path -> PEX_EXTRA_SYS_PATH which is needed to have the build see our now in-tree build backend.